makefiles/test.mk

```diff
@@ -12,7 +12,7 @@ e2e-nakama:
 		cd $(ROOT_DIR); \
 	)
 
-	@docker compose up --build --abort-on-container-exit --exit-code-from test_nakama --attach test_nakama
+	@docker compose up --build game nakama test_nakama --abort-on-container-exit --exit-code-from test_nakama --attach test_nakama
```

Comment:
this speeds up the e2e by a TONNNNN. without this line, every single service defined in the compose file gets invoked - meaning the chain + celestia client runs as well 🙈